frontend/src/metabase/redux/downloads.ts

```diff
@@ -31,7 +31,7 @@ export interface DownloadQueryResultsOpts {
   enablePivot?: boolean;
   dashboardId?: DashboardId;
   dashcardId?: DashCardId;
-  uuid?: string;
+  uuid?: EntityUuid | null;
```

Comment:
Was the type wrong, or we actually change the type signature to be nullable?

Reply:
I make it the same as in other places for the same prop